content/docs/user-guide/platforms/android/generating_android_project_windows.md

```diff
@@ -0,0 +1,139 @@
+{{< /note >}}
+
+
+**Step 1. Generate the cmake build directory for windows / vs2019 (for the android assets)**
+
+```
+cmake -B %O3DE_BUILD_ROOT%\windows -S %O3DE_ENGINE_PATH% -G"Visual Studio 16 2019" -DLY_PROJECTS=%O3DE_PROJECT_PATH% -DLY_UNITY_BUILD=ON
+```
+
+**Step 2. Build the Asset Processor Batch project in profile mode to process the assets**
+
+```
+cmake --build %O3DE_BUILD_ROOT%\windows --config profile --target AssetProcessorBatch -- /m /nologo
+```
+
+{{< note >}}
+This step will only build 'AssetProcessorBatch' and the necessary dependent modules. You can alternatively build the entire suite of tools, libraries, etc for windows by replacing `AssetProcessorBatch` with `ALL_BUILD`, or alternatively build from Visual Studio directly.
+{{< /note >}}
+
+**Step 3. Run Asset Processor Batch for android and Atom Sample Viewer**
+
+```
+%O3DE_BUILD_ROOT%\windows\bin\profile\AssetProcessorBatch.exe  --platforms=android --project-path %O3DE_PROJECT_PATH%
+```
+
+**Step 4. Generate a key store to be able to self-sign an APK**
+
+```
+keytool -genkey -keystore %O3DE_ANDROID_SIGNCONFIG_FILE% -storepass %O3DE_ANDROID_SIGNCONFIG_STORE_PASSWORD% -alias %O3DE_ANDROID_SIGNCONFIG_KEY_ALIAS% -keypass %O3DE_ANDROID_SIGNCONFIG_KEY_PASSWORD% -keyalg RSA -keysize %O3DE_ANDROID_SIGNCONFIG_KEY_SIZE% -validity %O3DE_ANDROID_SIGNCONFIG_VALIDITY_DAYS% -dname %O3DE_ANDROID_DN%
+```
+
+{{< note >}}
+Even though a signing key is required for the project in order to deploy, it does not necessary to provide this at project creation. This can also be achieved through Android Studio after project creation. If this is the case, then the generate android command in the following step can omit all of the --signconfig-* arguments, and the key store settings can be created through Android Studio after opening the generated android project.
+{{< /note >}}
+
+
+**Step 5. Generate the android project**
+
+```
+%O3DE_ENGINE_PATH%\python\python.cmd %O3DE_ENGINE_PATH%\cmake\Tools\Platform\Android\generate_android_project.py --engine-root %O3DE_ENGINE_PATH% --project-path %O3DE_PROJECT_PATH% --build-dir %O3DE_BUILD_ROOT%\android --third-party-path %USERPROFILE%\.o3de\3rdParty --android-sdk-path %O3DE_ANDROID_SDK_PATH% --android-ndk-version %O3DE_ANDROID_NDK_VERSION%  --android-sdk-platform %O3DE_ANDROID_SDK_API_LEVEL% --include-apk-assets --asset-mode %O3DE_ANDROID_ASSET_MODE% --signconfig-store-file %O3DE_ANDROID_SIGNCONFIG_FILE% --signconfig-store-password %O3DE_ANDROID_SIGNCONFIG_STORE_PASSWORD% --signconfig-key-alias %O3DE_ANDROID_SIGNCONFIG_KEY_ALIAS% --signconfig-key-password %O3DE_ANDROID_SIGNCONFIG_KEY_PASSWORD% --overwrite-existing
+```
+
+{{< note >}}
+At this point, you can alternatively open the folder %O3DE_BUILD_ROOT%\android directly in Android Studio and build the APK from there instead of using command line in the next step.
+{{< /note >}}
+
+**Step 6. Build the android project**
+
+```
+cd %O3DE_BUILD_ROOT%\android
+
+gradlew assembleProfile
+```
+
```

Comment:
- Make sure that note is indented so that it lines up underneath step 4. 
```suggestion

    {{< note >}}
Alternatively, instead of using command line, you can open the folder `%O3DE_BUILD_ROOT%\android` directly in Android Studio and build the APK from there.
    {{< /note >}}

```